readme.md

```diff
@@ -336,6 +336,7 @@
 ## Gaming
 
 - [Game Development](https://github.com/ellisonleao/magictools#readme)
+- [Game Networking](https://github.com/MFatihMAR/Awesome-Game-Networking#readme) - Network programming for multiplayer online games.
```

Comment:
I may or may not remove the description if you think it's a little bit long.

Reply:
It's fine to keep it.